cinema/models.py

```diff
@@ -0,0 +1,10 @@
+from django.db import models
+
+
+class Movie(models.Model):
+    title = models.CharField(max_length=255)
+    description = models.CharField(max_length=255, null=True)
```

Comment:
it's better to use TextField because you cannot be sure that description will fit in 255 characters, then this field can be nullable and no need to use required=False in serializer